### PR TITLE
Merge branch 'main' of https://github.com/kanjitp/RoadToHackathon int…

### PR DESCRIPTION
**Pull request by pojchrpol**
…o main

# Conflicts:
#	Java folder/HelloWorld.java

### CODE REVIEW

**Comment by pojchrpol** [comment]
gu pol

**Comment by mickJirapat** [comment]
what is this pol

**Comment by kanjitp** [comment]
Woah conflict omg

File: Java folder/HelloWorld.java [hunk 1]
```diff
@@ -7,6 +7,7 @@ public static void main(String[] args) {
         System.out.println("sawasdee");
         System.out.println("sawasdee");
         System.out.println("sawasdee");
+        System.out.println("gu,pol")
```
Comment (mickJirapat) [inline Java folder/HelloWorld.java:10]: Gumick

Comment (kanjitp) [inline Java folder/HelloWorld.java:10]: gu ryu

Comment (kanjitp) [reply Java folder/HelloWorld.java:10]: Bro you forgot semicolon

**Review by mickJirapat — COMMENTED** [review]
Good job

File: Java folder/HelloWorld.java [hunk 2]
```diff
@@ -7,6 +7,7 @@ public static void main(String[] args) {
         System.out.println("sawasdee");
         System.out.println("sawasdee");
         System.out.println("sawasdee");
+        System.out.println("gu,pol");
```
Comment (kanjitp) [inline Java folder/HelloWorld.java:10]: Nice

Comment (mickJirapat) [reply Java folder/HelloWorld.java:10]: very nice

Comment (kanjitp) [inline Java folder/HelloWorld.java:10]: I will merge now

**Review by kanjitp — APPROVED** [review]

**Pull request merged**